app/_landing_pages/gateway/deployment-topologies.yaml

```
@@ -8,12 +8,224 @@ rows:
+          
+  - header:
+      type: h2
+      text: "Hybrid mode"
+    columns:
+      - blocks:
+          - type: mermaid
+            config:
+              diagram: |
+                flowchart TD
+                A(Dev Portal &bull; Gateway Manager &bull; Analytics &bull; Service Catalog)
+                B(<img src="/assets/logos/KogoBlue.svg" style="max-height:20px"> Control Plane \n #40;Kong Gateway instance#41;)
+                B2(<img src="/assets/logos/KogoBlue.svg" style="max-height:20px"> Control Plane \n #40;Kong Gateway instance#41;)
+                C(<img src="/assets/logos/KogoBlue.svg" style="max-height:20px"> Data Plane 1\n #40;Kong Gateway instance#41;)
+                D(<img src="/assets/logos/KogoBlue.svg" style="max-height:20px"> Data Plane 2\n #40;Kong Gateway instance#41;)
+                E(<img src="/assets/logos/KogoBlue.svg" style="max-height:20px"> Data Plane 3\n #40;Kong Gateway instance#41;)
+                subgraph id1 [Konnect]
+                A --- B & B2
+                end
+                id1 --Kong proxy 
+                configuration---> id2 & id3
+                subgraph id2 [Kong-managed cloud node]
+                C
+                end
+                subgraph id3 [Self-managed local and cloud nodes]
+                D
+                E
+                end
+                style id1 stroke-dasharray:3,rx:10,ry:10
+                style id2 stroke-dasharray:3,rx:10,ry:10
+                style id3 stroke-dasharray:3,rx:10,ry:10
+                style B stroke:none,fill:#0E44A2,color:#fff
+                style B2 stroke:none,fill:#0E44A2,color:#fff
+      - blocks:
+          - type: structured_text
+            config:
+              blocks:
+                - type: text
+                  text: |
+                    In hybrid mode, {{site.base_gateway}} nodes in a cluster are split into two roles: Control Plane
+                    (CP), where configuration is managed, and data
+                    plane (DP), which serves traffic for the proxy.
+
+                    Hybrid mode deployments have the following benefits:
+
+                    * Users can deploy groups of Data Planes in different data centers, geographies, or zones without needing a local clustered database for each DP group.
+                    * The availability of the database does not affect the availability of the Data Planes. If a Control Plane is offline, Data Planes will run using their last known configuration.
+                    * Drastically reduces the amount of traffic to and from the database, since only CP nodes need a direct connection to the database.
+                    * If one of the DP nodes is compromised, an attacker won’t be able to affect other nodes in the {{site.base_gateway}} cluster.
+
+                    **Works on:** {{site.konnect_short_name}}, {{site.base_gateway}}
```

```suggestion
                    **Works on:** {{site.konnect_short_name}}, Self-hosted
```

Kong Gateway is a product, not a hosting method